Doppler.ContactPolicies.Data.Access/Repositories/ContactPoliciesSettings/ContactPoliciesSettingsRepository.cs

```diff
@@ -0,0 +1,58 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Dapper;
+using Doppler.ContactPolicies.Data.Access.Core;
+using Doppler.ContactPolicies.Data.Access.Entities;
+
+namespace Doppler.ContactPolicies.Data.Access.Repositories.ContactPoliciesSettings
+{
+    public class ContactPoliciesSettingsRepository : IContactPoliciesSettingsRepository
+    {
+        private readonly IDatabaseConnectionFactory _databaseConnectionFactory;
+
+        public ContactPoliciesSettingsRepository(IDatabaseConnectionFactory databaseConnectionFactory)
+        {
+            _databaseConnectionFactory = databaseConnectionFactory;
+        }
+
+        #region public methods
+
+        public async Task<Entities.ContactPoliciesSettings> GetContactPoliciesSettingsAsync(string accountName)
+        {
+            using var connection = await _databaseConnectionFactory.GetConnection();
+            const string query =
+                @"select cp.IdUser, cp.Enabled [EnabledExcludedSubscribersList], cp.Active, cp.Interval [IntervalInDays], cp.Amount [EmailsAmountByInterval], u.Email [AccountName]
+                from [User] u
+                left join [UserShippingLimit] cp on u.IdUser = cp.IdUser
+                where u.Email = @Email;
```

Comment:
Creo que para simplificar estaría bueno directamente ignorar las filas con Enable diferente de 1 de esta manera:

```suggestion
                @"select cp.IdUser, cp.Active, cp.Interval [IntervalInDays], cp.Amount [EmailsAmountByInterval], u.Email [AccountName]
                from [User] u
                left join [UserShippingLimit] cp on u.IdUser = cp.IdUser and cp.Enabled = 1
                where u.Email = @Email;
```